dask/widgets/templates/highlevelgraph.html.j2

```diff
@@ -15,7 +15,7 @@
         <div style="margin-left: 64px;">
             <h3 style="margin-bottom: 0px;">HighLevelGraph</h3>
             <p style="color: var(--jp-ui-font-color2, #5D5851); margin-bottom:0px;">
-                {{ type }} with {{ layers | length }} layers.
+                {{ type }} with {{ layers | length }} layers and {{ n_outputs }} outputs.
```

Comment:
I'm finding the term "outputs" to be a bit ambiguous. As used here, it's all the "output" keys for each individual layer. But I could also imagine it being meaning the number of output keys for the *last* layer. That is to say, the number of outputs for the whole computation, discarding intermediate keys.

We could, of course, include both. But since users are also able to inspect the number of outputs in the last layer already, maybe we could just say something like "`and {{ n_outputs }} keys from all layers`". Or maybe it's okay as-is. What do you think @kori73 ?

Reply:
I totally agree that `outputs` sounds like the output keys for the last layer. `keys from all layers` would be much less ambiguous.